### PR TITLE
Corrected import and added IP on which to listen to devel server.

### PR DESCRIPTION
**Pull request by rbeneyt**

### CODE REVIEW

**Review by fergalwalsh — CHANGES_REQUESTED** [review]
Thanks for the fix and enhancement. Both issues bothered me recently but I didn't get around to fixing them. See the inline comments for some suggested edits.

File: pico/server.py [hunk 1]
```diff
@@ -29,4 +30,10 @@ def run_app(app, ip='127.0.0.1', port=4242, use_debugger=True, use_reloader=True
...
             module_name += ':app'
         app = import_string(module_name)
-    run_app(app)
+
+        try:
+          server_ip = sys.argv[2]
+        except Exception:
+          server_ip = '127.0.0.1'
+
+    run_app(app, server_ip)
```
[the remark this answers is not in the document]
Comment (rbeneyt) [reply pico/server.py]: Hi Fergal, yep that's better. I'll ad the port in another request.

File: pico/server.py [hunk 2]
```diff
@@ -3,7 +3,8 @@
 import socket
 
 from werkzeug.serving import run_simple
-from werkzeug.wsgi import SharedDataMiddleware
+#from werkzeug.wsgi import SharedDataMiddleware
```
[the remark this answers is not in the document]
Comment (rbeneyt) [reply pico/server.py]: Ok, I'm not sure about how to do it, first time using github, so I'll submit another pull request with all the changes.

[the remark this answers is not in the document]
Comment (rbeneyt) [reply pico/server.py]: Added 2 commits with the changes, below.

**Review by fergalwalsh — APPROVED** [review]
That's great. Thank you.

**Comment by fergalwalsh** [comment]
Congrats on your first PR 🎉

**Comment by fergalwalsh** [comment]
FYI this is merged to master but I have not yet pushed a new release to PyPI. In the meantime you can install it directly from git using `pip install git+https://github.com/fergalwalsh/pico.git@master`

**Comment by StefanoSega** [comment]
@fergalwalsh  it doesn't work for me with Werkzeug 0.14.1, it works btw with Werkzeug 0.16.1

**Comment by fergalwalsh** [comment]
Thanks @StefanoSega, I have bumped the requirement now. It is in the master branch.

**Pull request merged**